draft-bbs-signatures.md

```diff
@@ -387,15 +387,14 @@ Procedure:
 This operation computes a deterministic signature from a secret key (SK) and optionally over a header and or a vector of messages.
 
 ```
-signature = Sign(SK, PK, header, (msg_1,..., msg_L), (H_1,..., H_L))
+signature = Sign(SK, PK, header, [(H_1, msg_1), ..., (H_L, msg_L)])
 
 Inputs:
 
 - SK (REQUIRED), an octet string of the form outputted by the KeyGen operation.
 - PK (REQUIRED), an octet string of the form outputted by the SkToPk operation provided the above SK as input.
 - header (OPTIONAL), an octet string containing context and application specific information. If not supplied, it defaults to an empty string.
-- msg_1,..., msg_L (OPTIONAL), a vector of octet strings.
-- H_1,..., H_L (OPTIONAL), points of G1. Generators used to commit each message.
+- [(H_1, msg_1),..., (H_L, msg_L)] (OPTIONAL), a list of generators and message pairs corresponding to messages to be signed. Generators must be G1 points, and the messages must be non-zero scalars mod r. If not supplied, it will default to the empty array ("[]"). The list MUST be ordered based on the index of the generators.
```

Comment:
```suggestion
- [(H_1, msg_1),..., (H_L, msg_L)] (OPTIONAL), a list of generators and message pairs corresponding to messages to be signed. Generators must be G1 points, and the messages must be non-zero scalars mod r. If not supplied, it will default to the empty array ("[]").
```

Do we need the last sentence here?

Reply:
Indeed we don't. Thanks! Updated.